packages/node/src/login/oidc/OidcLoginHandler.ts

```diff
@@ -67,24 +69,26 @@ export default class OidcLoginHandler implements ILoginHandler {
   ) {}
 
   async canHandle(options: ILoginOptions): Promise<boolean> {
-    return hasIssuer(options) && hasRedirectUrl(options);
+    return hasIssuer(options);
   }
 
-  async handle(options: ILoginOptions): Promise<void> {
+  async handle(
+    options: ILoginOptions
+  ): Promise<(ISessionInfo & { fetch: typeof fetch }) | undefined> {
```

Comment:
Ditto - i.e. isn't this just the type `LoginResult`...?

Reply:
Fixed in 6ee8aac0c890e90d7e3af0fdf265de1327bb0a53